api/grafana/grafana.go

```diff
@@ -2,12 +2,15 @@ package grafana
 
 import (
 	"context"
+	"errors"
 	"net/url"
 	"strconv"
 
 	"github.com/grafana/detect-angular-dashboards/api"
 )
 
+var errUnknownAngularStatus = errors.New("could not determine if plugin is angular or not")
```

Comment:
nit: I'd change the error message to say that angular info is missing and use GCOM instead.